src/modules/navigator/mission_base.h

```diff
@@ -328,6 +328,7 @@ class MissionBase : public MissionBlock, public ModuleParams
 	mission_s _mission;					/**< Currently active mission*/
 	float _mission_init_climb_altitude_amsl{NAN}; 		/**< altitude AMSL the vehicle will climb to when mission starts */
 	int _inactivation_index{-1}; // index of mission item at which the mission was paused. Used to resume survey missions at previous waypoint to not lose images.
+	int _replay_index{-1};					/**< Index of the mission item that will bring the drone back to a mission waypoint */
```

Comment:
```suggestion
	int _replay_index{-1};					/**< Index of the mission item that will bring the vehicle back to a mission waypoint */
```
Let's stay system agnostic.